docs/docs/0.5.0/policies/mutual-tls.md

```diff
@@ -1,167 +1,229 @@
+:::
+
+::: tab "Universal"
+```yaml
+type: Mesh
+name: default
+mtls:
+  enabledBackend: ca-1
+  backends:
     - name: ca-1
-      type: builtin # use Builtin CA (a unique Root CA certificate will be generated automatically)
+      type: builtin
+      dpCert:
+        rotation:
+          expiration: 24h
+      conf:
+        caCert:
+          RSAbits: 2048
+          expiration: 10y
 ```
 
-You can apply this configuration with `kubectl apply -f [file-path]`.
+You will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/0.5.0/documentation/http-api).
+:::
+::::
+
+A few considerations:
+
+* The `dpCert` configuration determines how often Kuma should automatically rotate the certificates assigned to every data plane proxy.
+* The `caCert` configuration determines a few properties that Kuma will use when auto-generating the CA root certificate.
 
-### Provided CA
+### Storage of Secrets
 
-In some cases users might need to opt out of auto-generated Root CA certificates, e.g. to stay compliant with internal company policies.
+When using a `builtin` backend Kuma automatically generates a root CA certificate and key that are being stored as a Kuma [Secret resource](/docs/0.5.0/documentation/secrets) with the following name:
 
-To that end, Kuma supports another type of CA - `provided` CA.
+* `{mesh name}.ca-builtin-cert-{backend name}` for the certificate
+* `{mesh name}.ca-builtin-key-{backend name}` for the key
 
-Like the name implies, a user will have to provide a custom Root CA certificate and take a responsibility for managing its lifecycle.
+On Kubernetes, Kuma secrets are being stored in the `kuma-system` namespace, while on Universal they are being stored in the underlying [backend](/docs/0.5.0/documentation/backends) configured in `kuma-cp`.
 
-A complete workflow of changing CA type from `builtin` to `provided` looks the following way:
+You can retrieve the secrets via `kumactl` on both Universal and Kubernetes, or via `kubectl` on Kubernetes only:
 
-1. A user must first upload a custom Root CA certificate using `kumactl manage ca provided` command, e.g.
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "kumactl"
 
-   ```shell
-   kumactl manage ca provided certificates add --mesh demo --cert-file demo.root_ca.crt --key-file demo.root_ca.key
-   ```
+The following command can be executed on any Kuma backend:
 
-   ::: warning
-   This is a security-sensitive opreation, which implies very restrictive default settings.
+```sh
+$ kumactl get secrets [-m MESH]
+MESH      NAME                           AGE
+default   default.ca-builtin-cert-ca-1   1m
+default   default.ca-builtin-key-ca-1    1m
+```
+:::
+::: tab "kubectl"
 
-   If you are just Getting Started with Kuma, run `kumactl manage ca provided` commands on the same machine where `kuma-cp` is running.
-   :::
+The following command can be executed only on Kubernetes:
+
+```sh
+$ kubectl get secrets -n kuma-system
```

Comment:
We need to filter by type `system.kuma.io/secret` otherwise it will show other secrets that are for CP like TLS certs for admission server etc.

Reply:
kubectl get secrets -n kuma-system --field-selector='type=system.kuma.io/secret'